WorkloadManagementSystem/Client/Matcher.py

```diff
@@ -68,11 +68,11 @@ def selectJob( self, resourceDescription, credDict ):
     result = self.tqDB.matchAndGetJob( resourceDict, negativeCond = negativeCond )
 
     if not result['OK']:
-      return result
+      raise RuntimeError( result['Message'] )
     result = result['Value']
     if not result['matchFound']:
       self.log.info( "No match found" )
-      raise RuntimeError( "No match found" )
+      return {}
```

Comment:
Ah, GitHub did not show the MatcherHandler changes, may be refresh page problem. So, it is OK then